src/txmempool.cpp

```diff
@@ -870,8 +870,13 @@ void CTxMemPool::PrioritiseTransaction(const uint256& hash, const CAmount& nFeeD
             }
             ++nTransactionsUpdated;
         }
+        if (delta == 0) {
+            mapDeltas.erase(hash);
+            LogPrintf("PrioritiseTransaction: %s delta cleared\n", hash.ToString());
+        } else {
+            LogPrintf("PrioritiseTransaction: %s fee += %s\n", hash.ToString(), FormatMoney(nFeeDelta));
```

Comment:
Shouldn't these messages be returned as an RPC result?

Shouldn't the message here also show the resulting total `delta`, and perhaps whether the tx was already in the mempool (`nTransactionsUpdated > 0`)?